include/aa_params.hpp

```diff
@@ -78,6 +78,11 @@ namespace astroaccelerate {
 #define DIT_YSTEP 2
 #define DIT_ELEMENTS_PER_THREAD 4
 
+//experimental clustering filter
+#define PPF_PEAKS_PER_BLOCK 10
+#define PPF_DPB 128
+#define PPF_SIZE_SEARCH 15
```

Comment:
I would change PPF_SIZE_SEARCH to PPF_SEARCH_RANGE_IN_MS to indicate that it is in ms maybe add comment as well.

Reply:
Done